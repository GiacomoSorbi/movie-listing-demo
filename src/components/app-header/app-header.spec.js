import React from 'react'
import { mount } from 'enzyme'
import toJson from 'enzyme-to-json'

import AddHeader from './'

describe('AddHeader should', () => {
  it('exist', () => {
    expect(AddHeader).toBeDefined()
  })

  it('have a consistent structure', () => {
    const wrapper = mount(<AddHeader />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
