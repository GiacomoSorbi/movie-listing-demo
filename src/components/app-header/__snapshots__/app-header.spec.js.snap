// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddHeader should have a consistent structure 1`] = `
<AppHeader>
  <header
    className="app-header"
  >
    <div
      className="app-logo"
    />
    <h1>
      Currently airing movies
    </h1>
    <div
      className="app-logo"
    />
  </header>
</AppHeader>
`;
